### DISCUSSION TITLE
Capytaine installation

### DISCUSSION THREAD

**Discussion by Ekaterinauihuihuihuihuihuhh** [capytaine/capytaine#276] [General]
Hi all!

I install python 3.11 and Conda 4.12.0

When I use command: conda install -c conda-forge capytaine
Conda write me:

*****************************************************************************************************************
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - capytaine

Current channels:

  - https://conda.anaconda.org/conda-forge/win-32
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/win-32
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-32
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-32
  - https://repo.anaconda.com/pkgs/msys2/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

***********************************************************************************************************************

I dont understand what can I do in order to install Capytaine(
Could you please to help me?

**Comment by mancellin** [comment]
Given the `win-32` in the message from conda, I'm guessing you are using a 32-bit version of Windows.

There are no precompiled binaries for Capytaine for this platform.
The documentation should be clarified to mention *64-bit* Windows.

If there is a need from many users, I'll look into building also 32-bit packages.

**Reply by accient** [reply]
@mancellin 
Hello, I'm trying to install capytaine1.5, But with the following command the v1.4 will be installed by default.
![image](https://user-images.githubusercontent.com/37647393/222999559-1eae63aa-b014-4949-b58d-edc8a39e577a.png)
 When I specify the version=1.5, it says"the packages are not available from current channels" and recommends me to search on "https://anaconda.or".
![image](https://user-images.githubusercontent.com/37647393/223000724-f1b77f9e-b446-4a27-9adb-160631fa70a5.png)
![image](https://user-images.githubusercontent.com/37647393/223000809-fca9a5f6-5fcd-48a6-a22a-611273c37e31.png)
With these command, I can still get v1.4. Woud you please give me some suggestion?

**Reply by mancellin** [reply]
@accient Does the solution of #299 (creating a new clean environment for Capytaine) also works for you?

**Reply by accient** [reply]
@mancellin Yeah, it works now. Thanks for your help!
